Packs/Twitter/Integrations/Twitterv2/Twitterv2.py

```diff
@@ -0,0 +1,388 @@
+                                                       'edit_history_tweet_ids': data_item.get('id', None)}
+        list_dict_response.append(remove_empty_elements(dict_to_append))
+    return list_dict_response
+
+
+''' COMMAND FUNCTIONS '''
+
+
+def test_module(client: Client) -> str:
+    """Tests API connectivity and authentication'
+
+    Returning 'ok' indicates that the integration works like it is supposed to.
+    Connection to the service is successful.
+    Raises exceptions if something goes wrong.
+
+    :type client: ``Client``
+    :param Client: client to use
+
+    :return: 'ok' if test passed, anything else will fail the test.
+    :rtype: ``str``
+    """
+
+    message: str = ''
+    client.tweet_search('Twitter', None, None, 10, None)
+    message = 'ok'
+    return message
+
+
+def date_to_iso_format(date: str) -> str:
+    """ Retrieves date string or relational expression to iso format date.
+        Args:
+            date: str - date or relational expression.
+        Returns:
+            A str in ISO format.
+    """
+    date = dateparser.parse(date)
+    date = date.strftime("%Y-%m-%dT%H:%M:%SZ") if date else ''
+    return date
+
+
+def create_human_readable_search(dict_list: list[dict]) -> list[dict]:
+    """Gets list of dictionaries with a dictionaries inside for example [{key1: {key2: value}}, {key1: {key2: value}}]
+    and changes it to a list of dictionaries without any dictionaries inside
+    for example [{key1.key2: value}, {key1.key2: value}].
+    Args:
+        dict_list: list[dict] -  A list of dictionaries.
+    Returns:
+        A header string.
+    """
+    list_dict_response: list = []
+    for dict_value in dict_list:
+        list_dict_response.append(
+            {
+                "Tweet ID": dict_value.get("id", {}),
+                "Text": dict_value.get("text", ""),
+                "Created At": dict_value.get("created_at", {}),
+                "Author Name": dict_value.get("author", {}).get("name"),
+                "Author Username": dict_value.get("author", {}).get("username"),
+                "Likes Count": dict_value.get("public_metrics", {}).get("like_count"),
+                "Attachments URL": [
+                    item.get("url", "") for item in dict_value.get("media", [])
+                ]
+                if dict_value.get("media", [])
+                else dict_value.get("media", []),
+            }
+        )
+    return list_dict_response
+
+
+def create_human_readable_user(dict_list: list[dict]) -> list[dict]:
+    """ Gets list of dictionaries with a dictionaries inside for example [{key1: {key2: value}}, {key1: {key2: value}}]
+        and changes it to a list of dictionaries without any dictionaries inside
+        for example [{key1.key2: value}, {key1.key2: value}].
+        Args:
+            dict_list: list[dict] -  A list of dictionaries.
+        Returns:
+            A header string.
+    """
+    list_dict_response: list = []
```

Comment:
No need for the helper function - do this directly in the command, it relatively short